github/data_source_github_user_external_identity_test.go

```diff
@@ -0,0 +1,51 @@
+package github
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccGithubUserExternalIdentity(t *testing.T) {
```

Comment:
I wasn't sure how to test this, since it depend on the GitHub Organization having SAML configured, as well as users with connected identities. I've manually tested it with my org, but if there is a good way to test this let me know

Reply:
I don't know a better way to go about it, sorry. Manually testing in your org is good for now.

Reply:
Then it is what it is